movie-app/src/Components/MovieCard.js

```diff
@@ -1,14 +1,34 @@
+
+import {useState} from 'react';
+
 const MovieCard = ({movie}) => {
+  const moviePoster = {
+    backgroundImage: `url(${movie.Poster})`,
+  }
+
+  const [cardHover, setCardHover] = useState(false);
+// when true, apply show info animation
+  const showInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
+
+  const hideInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
```

Comment:
Both of these functions are the same.  You can combine them as a `toggleInfoOnHover` function.